rdf-star-cg-spec.html

```diff
@@ -829,9 +829,101 @@ <h2>SPARQL* Update</h2>
+      </ul>
+      </p>
+
+      <p>Given an <a>RDF* graph</a> |G|, the following steps transform it into an <a>RDF graph</a> that we call |unstar|(|G|).</p>
+
+      <ol>
+        <li>Pick an <a>RDF* triple</a> (|s|, |p|, |o|) in the <a>constituents</a> of |G| such that neither |s| nor |o| is an <a>embedded triple</a>.</li>
+        <li>Mint a fresh blank node |b|, and replace by |b| all occurrences of (|s|, |p|, |o|) in the subject or object position of an <a>asserted</a> or <a>embedded triple</a> of |G|.</li>
+        <li>Add the following <a>asserted triples</a> to |G|:<ul>
+          <li>(|b|, |ST|, |s|)</li>
+          <li>(|b|, |PT|, |p|)</li>
+          <li>(|b|, |OT|, |o|) unless |o| is an <a>ill-typed</a> literal</li>
+          <li>(|b|, |SS|, |L|(|s|)) unless |s| is <a>blank node</a></li>
+          <li>(|b|, |PS|, |L|(|p|))</li>
+          <li>(|b|, |OS|, |L|(|o|)) unless |o| is <a>blank node</a></li>
+        </ul></li>
+        <li>Repeat the steps above until there are no embedded triples left in |G|.</li>
+      </ol>
+      <p>After these steps, |unstar|(|G|) is an <a>RDF graph</a>, as it contains no embedded triples. Note that if |G| was already an <a>RDF graph</a>, then |unstar|(|G|) = |G|.</p>
+
+    </section>
+
+    <section>
+      <h2>Entailment of <a>RDF* graphs</a></h2>
+
+      <p>Following [[[RDF11-MT]]], we extend the notions of <a>satisfiability</a> and <a>entailment</a> for <a>RDF* graphs</a>. Given two <a>RDF* graphs</a> |G| and |H|:<p>
+      <ul>
+        <li>We say that |G| is (simply) <a>satisfiable</a> (resp. <a>unsatisfiable</a>) if and only if |unstar|(|G|) is (simply) <a>satisfiable</a> (resp. <a>unsatisfiable</a>).</li>
+        <li>We say that |H| is (simply) <a>entailed</a> by (resp. <a>equivalent</a> to) |G| if and only if |unstar|(|H|) is (simply) <a>entailed</a> by (resp. <a>equivalent</a> to) |unstar|(|G|).</li>
+        <li>Other notions of <a>satisfiability</a> and <a>entailment</a>, such as <a data-cite="RDF11-MT#rdf-entailment">RDF entailment</a> or <a data-cite="RDF11-MT#rdfs-entailment">RDFS entailment</a>, can be extended in the same way for <a>RDF* graphs</a>.</li>
+      </ul>
+    </section>
 
-    <div class="issue" data-number="81">one proposal on the table</div>
+    <section class="informative">
+      <h2>Remarks</h2>
+
+      <section>
+        <h2>Combining RDF* graphs</h2>
+
+        <p>Care must be taken when <a>RDF* graphs</a> are combined through union or <a>merging</a>. Given two <a>RDF* graphs</a> |G| and |H|, it may be the case that |unstar|(|G| ∪ |H|) ≠ |unstar|(|G|) ∪ |unstar|(|H|). More precisely, if |G| and |H| contain the same <a>embedded triple</a>, this triple will be mapped to a single blank node in |unstar|(|G| ∪ |H|), but in two potentially different blank nodes in |unstar|(|G|) ∪ |unstar|(|H|). These blank nodes will need to be unified in order to get the correct entailments.</p>
```

Comment:
```suggestion
        <p>Care must be taken when <a>RDF* graphs</a> are combined through union or <a>merging</a> if those graphs have been mapped beforehand to <a>RDF graphs</a>. Given two <a>RDF* graphs</a> |G| and |H|, it may be the case that |unstar|(|G| ∪ |H|) ≠ |unstar|(|G|) ∪ |unstar|(|H|).More precisely, if |G| and |H| contain the same <a>embedded triple</a>, this triple will be mapped to a single blank node in |unstar|(|G| ∪ |H|), but in two potentially different blank nodes in |unstar|(|G|) ∪ |unstar|(|H|). These blank nodes will need to be unified in order to get the correct entailments.</p>
```
@pfps would that address [your comment](https://github.com/w3c/rdf-star/pull/88#issuecomment-778285079)